addons/Dockerfile

```diff
@@ -7,26 +7,30 @@
 # Build from publicly reachable source by default, but allow people to re-build images on
 # top of their own trusted images.
 ARG BUILDER_BASE_IMAGE=golang:1.18
-
+ARG DISTROLESS_BASE_IMAGE=gcr.io/distroless/static:nonroot
 # Build the manager binary
 FROM $BUILDER_BASE_IMAGE as builder
 WORKDIR /workspace
 # Copy the go source
 COPY ./ ./
 
-
+ARG GOSUMDB
+ARG GOPROXY
+ENV GOSUMDB=$GOSUMDB
+ENV GOPROXY=$GOPROXY
```

Comment:
this can use a comment like: setting values of default golang environment variable using build time arguments which will be used while building go binary

Reply:
added comment